portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,33 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  static ArrayList<String> stringList = new ArrayList<String>();
```

Comment:
Is this intended to be empty?

Reply:
Yes. Later the user comments will be added as list elements. I'll leave a comment here to avoid the confusion.

Reply:
I think the user comments will be fetched from the datastore, so it is not going to be a class variable write?

Reply:
That's the next step in the tutorial which I am still working on:D